internal/pkg/v2/infrastructure/redis/device_profile.go

```diff
@@ -18,7 +18,10 @@ import (
 	"github.com/gomodule/redigo/redis"
 )
 
-const DeviceProfileCollection = "v2:deviceProfile"
+const (
+	DeviceProfileCollection             = "v2:deviceProfile"
```

Comment:
Question , looking forward as we move forward and remove V1 code do we want the collection names prefixed with `v2`? If we do a `v3` that doesn't require revamp of DB layer then the `v2` prefix is strange. @cloudxxx8 , thoughts.

Reply:
I just discussed with Bruce today.  we plan to modify all the Redis key pre-fix. "v2" was for the experiment in Hanoi.
We plan to change to "md|dp" here to reduce the space consumption. "md" is the pre-fix for metadata for #2861

Reply:
Excellent. THX!